src/Model/Release/LibraryRelease.php

```diff
@@ -227,9 +234,65 @@ public function getPriorVersion($fallback = true)
     public function setPriorVersion(Version $version)
     {
         $this->priorVersion = $version;
+
+        // Clear the child prior versions known as this now might change
+        $this->childPriorVersions = null;
+
         return $this;
     }
 
+    /**
+     * Attempt to determine what prior versions of child modules were released by looking up the composer information
+     * for the prior release of this module
+     *
+     * @return array
+     */
+    public function getChildPriorVersions()
+    {
+        if ($this->childPriorVersions !== null) {
+            return $this->childPriorVersions;
+        }
+
+        $priorVersion = $this->getPriorVersion();
+
+        if (!$priorVersion) {
+            return [];
+        }
+
+        $composerData = $this->getLibrary()->getHistoryComposerData($priorVersion);
+        $childReleases = $this->getLibrary()->getChildren();
+        $childPriorVersions = [];
+
+        foreach ($childReleases as $child) {
+            $childName = $child->getName();
+            if (!isset($composerData['require'][$childName])) {
+                continue;
+            }
+
+            $constraint = $composerData['require'][$childName];
+
+            // Attempt to resolve a specific version from the constraint
+            if (!preg_match('/^\d+\.\d+\.\d+/', $constraint, $matches)) {
+                continue;
+            }
+
+            $childPriorVersions[$childName] = new Version($matches[0]);
```

Comment:
This doesn't work. Got a fix but I'm currently WIP on the other (draft) PR I have open right now. It shouldn't use the result of the `preg_match` here. I've ended up just using `Version::parse` above instead of a `preg_match` as that does it's own, more complex matching. If it doesn't parse then it continues.

Reply:
Ok I tried to fix and push but I don't have rights to the creative-commoners fork now :(